cli/tests/compiler_api_test.ts

```diff
@@ -292,3 +292,21 @@ Deno.test({
     assert(diagnostics[0].messageText.includes("This import is never used"));
   },
 });
+
+Deno.test({
+  name: `Deno.emit() - bundle supports iife`,
+  async fn() {
+    const { diagnostics, files } = await Deno.emit("/a.ts", {
+      bundle: "iife",
+      sources: {
+        "/a.ts": `import { b } from "./b.ts";
+          console.log(b);`,
+        "/b.ts": `export const b = "b";`,
+      },
+    });
+    assert(diagnostics);
+    assertEquals(diagnostics.length, 0);
+    assertEquals(Object.keys(files).length, 1);
+    assert(files["deno:///bundle.js"].startsWith("function() {\n"));
```

Comment:
Should it start with `(function() {\n`?

Reply:
I think I made a mistake.
I'll fix it and publish a new version. (fixer should be called after wrapping with iife, but it's not)

Reply:
I published a new version of `swc_bundler`.